<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="ecrmac-100510.local" ignored="2" name="sandbox" tests="6" failures="1" timestamp="03 Oct 2018 19:44:14 GMT" time="104.073" errors="0">
  <testcase name="@AfterMethod getResult" time="0.0" classname="reports.ExtentReportsClass">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at reports.ExtentReportsClass.getResult(ExtentReportsClass.java:37)
... Removed 28 stack frames]]>
    </failure>
  </testcase> <!-- @AfterMethod getResult -->
  <testcase name="@AfterTest endReport" time="0.0" classname="reports.ExtentReportsClass">
    <skipped/>
  </testcase> <!-- @AfterTest endReport -->
  <testcase name="loginComSucesso" time="16.984" classname="tests.Login"/>
  <testcase name="autenticaFalha" time="23.248" classname="tests.Login"/>
  <testcase name="autenticaFalha" time="17.078" classname="tests.Login"/>
  <testcase name="autenticaFalha" time="21.145" classname="tests.Login"/>
  <testcase name="autenticaFalha" time="24.709" classname="tests.Login"/>
  <testcase name="autenticainvoice" time="0.004" classname="tests.InvoiceTest">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at tests.InvoiceTest.autenticainvoice(InvoiceTest.java:25)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- autenticainvoice -->
  <testcase name="@BeforeTest startTest" time="0.0" classname="reports.ExtentReportsClass">
    <ignored/>
  </testcase> <!-- @BeforeTest startTest -->
  <testcase name="@AfterTest endReport" time="0.0" classname="reports.ExtentReportsClass">
    <ignored/>
  </testcase> <!-- @AfterTest endReport -->
</testsuite> <!-- sandbox -->
